### DISCUSSION TITLE
License period & file format

### DISCUSSION THREAD

**Discussion by fruttivendolo** [openaudible/openaudible#674] [General]
Ciao,
Before purchasing a license for OpenAudible I would appreciate a response to some issues that I have not been able to find an answer to.

Firstly, is an OpenAudible license, for Linux, for a limited period of time and if so for how long. Secondly, if Audible make some changes to the format of their audio book files are all current license holders entitled to an updated version of OpenAudible?

Best regards

**Comment by openaudible** [comment]
License is good forever, but free upgrades are for 1 year only. We've improved the description on our license page: https://openaudible.org/license
